src/actions/checkSolution.ts

```diff
@@ -107,8 +109,8 @@ export async function checkSolution(parameters: CheckSolutionParameters, runnerP
     const pacResult = await pac(...pacArgs);
     logger.log("CheckSolution Action Result: " + pacResult);
 
-    const files = glob.sync('**/*', { cwd: outputDirectory, absolute: true });
-    const artifactStoreName = validator.getInput(parameters.artifactStoreName) || 'CheckSolutionLogs';
+    const files: string[] = glob.sync('**/*', { cwd: outputDirectory, absolute: true }).filter((file: string) => !fs.lstatSync(file).isDirectory());
+    const artifactStoreName = validator.getInput(parameters.artifactStoreName) || 'CodeAnalysisLogs';
```

Comment:
Curious what this change to ```CodeAnalysisLogs``` from ```CheckSolutionLogs``` is about?

Reply:
Default support for the Sarif Sast Scans Tab tool - https://marketplace.visualstudio.com/items?itemName=sariftools.scans&ssr=false#overview when using build pipelines.